jetson/odrive_bridge/src/__main__.py

```diff
@@ -209,7 +209,7 @@ def connect(self):
 
         odrives = ["335D36623539", "335B36563539", "2066377F5753"]

```

Comment:
i would honestly look into making this a config thing

Reply:
sure, added to ToDo